og_ui/src/BundleFormAlter.php

```diff
@@ -123,12 +123,12 @@ protected function addGroupContent(array &$form, $form_state) {
     $target_types = [];
     $bundle_options = [];
     $all_group_bundles = Og::groupManager()->getAllGroupBundles();
-    foreach ($all_group_bundles as $group_entity_type => $bundles) {
+    foreach ($all_group_bundles as $entity_type => $bundles) {
```

Comment:
@RoySegall can you (once your body temperature goes below 39 degrees celsius) please revert the variable names back to the long form. This would make @chx a happier dev :wink: